iquip/apps/builder.py

```diff
@@ -118,38 +114,38 @@ class _NumberEntry(_BaseEntry):
     
     Attributes:
         argInfo: Each key and its value are:
-            default: The default value. If None, it is set to the min value.
+            (optional) default: The number value.
+              If not exist, the spinBox is set to the min value.
             unit: The unit of the value.
             scale: The scale factor that is multiplied to the number value.
             step: The step between values changed by the up and down button.
             min: The minimum value. (default=0.0)
             max: The maximum value. (default=99.99)
             ndecimals: The number of displayed decimals.
-            type: The type of the value. If "int", value() returns an integer value.
+            type: The type of the value.
+              If "int", value() returns an integer value.
+              Otherwise, it is regarded as a float value.
         spinBox: The spinbox showing the number value.
     
     TODO(BECATRUE): The operations of unit and scale will be concretized in Basic Runner project.
+    TODO(BECATRUE): Handling the case where the default doesn't exist and the min is None
```

Comment:
Umm.. It is entirely up to us.
In fact, the artiq-dashboard doesn't handle the default value when it is not given.

However, I want to handle it to prevent the non-default error.
So I planned to set it just to zero.

Reply:
Do you mean it depends on the implementation of `artiq-proxy`?

Reply:
Nope! In the artiq-dashboard, if the default value is not given, it left the empty entry.
In this case, if the user submits it without setting the value, it would result in the non-default error.

Thus, I want to handle entries of which default value is not given to prevent forgetting to set it from an user.